scipy/stats/stats.py

```diff
@@ -7372,63 +7373,59 @@ def kstest(rvs, cdf, args=(), N=20, alternative='two-sided', mode='auto'):
     Note that the alternative hypotheses describe the *CDFs* of the
     underlying distributions, not the observed values. For example,
     suppose x1 ~ F and x2 ~ G. If F(x) > G(x) for all x, the values in
-    x1 tend to be less than those in x2.      
+    x1 tend to be less than those in x2.
 
 
     Examples
     --------
     >>> from scipy import stats
+    >>> rng = np.random.default_rng()
 
     >>> x = np.linspace(-15, 15, 9)
     >>> stats.kstest(x, 'norm')
-    (0.44435602715924361, 0.038850142705171065)
+    KstestResult(statistic=0.444356027159..., pvalue=0.038850140086...)
 
-    >>> np.random.seed(987654321) # set random seed to get the same result
-    >>> stats.kstest(stats.norm.rvs(size=100), stats.norm.cdf)
-    (0.058352892479417884, 0.8653960860778898)
+    >>> stats.kstest(stats.norm.rvs(size=100, random_state=rng), stats.norm.cdf)
+    KstestResult(statistic=0.165471391799..., pvalue=0.007331283245...)
 
     The above lines are equivalent to:
 
-    >>> np.random.seed(987654321)
     >>> stats.kstest(stats.norm.rvs, 'norm', N=100)
-    (0.058352892479417884, 0.8653960860778898)
+    KstestResult(statistic=0.113810164200..., pvalue=0.138690052319...)  # may vary
```

Comment:
This line isn't equivalent any more since the above one used a `random_state`. Is there any way to pass it here as well?

Reply:
True, but apparently there is no way to pass a seed here. There is `args` but it's for the CDF and it's doing `*args`.